lib/perso.ml

```diff
@@ -2374,26 +2374,14 @@ and eval_item_field_var ell = function
       with Failure _ -> raise Not_found)
   | _ -> raise Not_found
 
-and eval_title_field_var conf base env (p, (cnt, name, title, places, dates))
+and eval_title_field_var conf base env (_p, (nth, name, title, places, dates))
     _loc = function
   | [ "is_first" ] ->
       VVbool
         (match get_env "first" env with Vbool x -> x | _ -> raise Not_found)
   | [ "is_main" ] -> (
-      let titles = nobility_titles_list conf base p in
-      let has_main =
-        List.fold_left
-          (fun acc (_, name, _, _, _) ->
-            acc || match name with Tmain -> true | _ -> false)
-          false titles
-      in
-      match name with
-      | Tmain -> VVbool true
-      | _ ->
-          if has_main then VVbool false
-          else
-            VVbool (match get_env "first" env with Vbool x -> x | _ -> false))
-  | [ "cnt" ] -> VVstring (string_of_int cnt)
+      match name with Tmain -> bool_val true | _ -> bool_val false)
```

Comment:
why was "is_main" more complicated before? What is "is_main"?

Reply:
on a déjà name! mais il manque effectivement le test de is_first. Je corrige ce point